pkgs/test_core/lib/src/runner/parse_metadata.dart

```diff
@@ -348,6 +350,8 @@ class _Parser {
 
   String? _findConstructorNameFromInstantiation(
       InstanceCreationExpression constructor, String className) {
+    // Replace name2 with name when bumping min analyzer dependency
+    // ignore: deprecated_member_use
```

Comment:
Why not doing it now?
When we will start removing old APIs from the analyzer, this will cause issues.
And than we will have to do one more iteration to clean this up.

Reply:
I think we should have at least 1 release which allows both analyzer versions so I'll publish first with the ignore and then fix it in a followup.